src/ext/property-panel/data.js

```diff
@@ -165,12 +165,8 @@ const getData = (env) =>
     ? {
         type: 'items',
         translation: 'Common.Data',
-        items: {
-          dataAssets: {
-            type: 'items',
-            component: 'data-assets-panel',
-          },
-        },
+        component: 'data-assets-panel',
```

Comment:
Why change the structure here? does this follow the property panel common structure? will this change affect how the data component will render?

Reply:
Yes it follows the property panel structure. No it wont. It will only affect the rendering of DataAssetPanel. This change is necessary to make the DataAssetPanel expand to 100% of the height of parent component.

Reply:
Ok, sounds good to me, i would like to have @veinfors opinion on this one.

Reply:
Looks good to me!